resolwe_bio/descriptors/dicty.yml

```diff
@@ -178,8 +178,8 @@
               value: DIF treatment
             - label: Percoll gradient (Prespore cells - Heavy fraction)
               value: Percoll gradient (Prespore cells - Heavy fraction)
-            - label: Percoll gradient (Prestalk cells - Light fraction) 
-              value: Percoll gradient (Prestalk cells - Light fraction) 
+            - label: Percoll gradient (Prestalk cells - Light fraction)
```

Comment:
Are these changes related to the new command? If not, put them in separate commit.